tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py

```diff
@@ -174,9 +174,13 @@ def _get_user_roles(self, username, language):
                 return ["instructor", f"instructor-{language}"]
             else:
                 roles = self.extra_get_user_roles(username, decoded_access_token)
-                if bool("{{SUPERSET_BLOCK_STUDENT_ACCESS}}") and not roles:
+                if roles:
+                    return roles
```

Comment:
I'm somewhat concerned about this logic. Having a `student` role returned here would override `SUPERSET_BLOCK_STUDENT_ACCESS`. Maybe not a big deal, but can we add a check for that and throw an exception if it happens?